packages/patternfly-3/patternfly-react/src/components/DualListSelector/components/DualListItem.js

```diff
@@ -70,7 +92,10 @@ DualListItem.defaultProps = {
   filterTerm: null,
   onChange: noop,
   side: null,
-  hidden: false
+  hidden: false,
+  disabled: false,
+  tooltipID: `dual-list-item-tooltip-${UUID()}`,
```

Comment:
This only runs once at module import. Not when the component is instantiated so if someone mounts multiple of these they will all share the same id.

Reply:
Wow thanks.. I didn't know that..
maybe set the ID by the item side+position which is unique enough

Reply:
What we've done in other components is to set the default to null then check for it and if it is not passed use:
```
const randomId = () => Date.now();
...
<Tooltip id={tooltipID || randomId()}>
```

Reply:
Tell me what do you think, added this:
```
const getTooltipID = () => {
    let uniqueTooltipID = `dual-list-item-tooltip-${side}`;
    if (parentPosition) {
      uniqueTooltipID += `-${parentPosition}`;
    }
    uniqueTooltipID += `-${position}`;
    return uniqueTooltipID;
  };
```
and then using it like this when the default `tooltipID` is `null`.
```
    <DualListItemTooltip text={tooltipText} id={tooltipID || getTooltipID()}>
```